code/game/machinery/cell_charger.dm

```diff
@@ -91,7 +91,7 @@
 		if(!charging || (stat & (BROKEN|NOPOWER)) || !anchored)
 			return
 
-		use_power(2000)		//this used to use CELLRATE, but CELLRATE is fucking awful. feel free to fix this properly!
-		charging.give(1750)	//inefficiency.
+		use_power(200)		//this used to use CELLRATE, but CELLRATE is fucking awful. feel free to fix this properly!
+		charging.give(175)	//inefficiency.

```

Comment:
It is nonsense, yes, but so is just about every single other thing involving the powergrid. The amount of silliness that is inherent in how it works is ridiculous, and a few power useage tweaks here and there aren't going to work to resolve the issue because of how powernets and APCs, at their core, work; it's why you see things like SMESes increasing total available power to the grid just by existing.

Case and point here; cells already require more energy to charge up than they contain, but powernet and APC functionality is not exactly as you imagine it to be.